kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -2722,15 +2722,24 @@ messages:
       {
          oTimer = $;
       }
-      else 
+      else
       {
-         if lastcall
+         if NOT Send(self,@IsLoggedOn)
          {
-            oTimer = CreateTimer(self,@EnchantmentTimer,time);
+            % Player is offline. Store time in frozen integer format.
+            oTimer = time;
```

Comment:
I was wondering if I ought to renamed `oTimer` to something generic, like `timer`?